front/app/api/idea_statuses/useAddIdeaStatus.test.ts

```diff
@@ -0,0 +1,64 @@
+import { renderHook, act } from '@testing-library/react-hooks';
+
+import useAddIdeaStatus from './useAddIdeaStatus';
+
+import { setupServer } from 'msw/node';
+import { rest } from 'msw';
+
+import createQueryClientWrapper from 'utils/testUtils/queryClientWrapper';
+import { ideaStatusesData } from './__mocks__/useIdeaStatuses';
+
+const apiPath = '*idea_statuses';
+
+const server = setupServer(
```

Comment:
For api custom hooks, we use mock service worker to mock the api requests and responses.